includes/admin/reporting/class-base-logs-list-table.php

```diff
@@ -250,7 +242,7 @@ public function downloads_filter() {
 		) );
 
 		if ( $downloads ) {
-			echo '<select name="download" id="edd-log-download-filter" class="edd-select-chosen ' . esc_attr( $rtl_class ) . '">';
+			echo '<select name="download" id="edd-log-download-filter">';
```

Comment:
Why remove `chosen` from this dropdown? Everywhere else we display a select for downloads/products, it uses `chosen`, although I see that this does not use the `product_dropdown` method, which is maybe another question. But presumably this could be rather a long list, so keeping `chosen` here seems like it would make sense.

Reply:
@robincornett I removed it because it doesn't have the AJAX searching built in. It could be a long list but it is long with Chosen enabled as well. 

I wavered on this one but I think having a accessible long list is better than an inaccesible long list. It should eventually be updated to support AJAX searching.

Reply:
I did not realize it didn't include the search--so it really wasn't using the script at all in the first place.

Reply:
@robincornett Chosen will show a "Search" field if there are more than X amount of items, but since the Downloads are listed alphabetically it's just as easy to select an option by typing the name and selecting with built in `<select>` behavior.